code/drasil-printers/lib/Language/Drasil/HTML/Print.hs

```diff
@@ -21,14 +21,14 @@ import qualified Language.Drasil as L
 
 import Language.Drasil.HTML.Monad (unPH)
 import Language.Drasil.HTML.Helpers (articleTitle, author, ba, body, bold,
-  caption, divTag, em, h, headTag, html, image, li, ol, pa,
+  caption, divTag, spanTag', em, h, headTag, html, image, li, ol, pa,
   paragraph, reflink, reflinkInfo, reflinkURI, refwrap, refwrap', sub, sup, table, td,
   th, title, tr, ul, dl, dd, BibFormatter(..))
 import Language.Drasil.HTML.CSS (linkCSS)
 
 import Language.Drasil.Config (StyleGuide(APA, MLA, Chicago), bibStyleH)
 import Language.Drasil.Printing.Import (makeDocument)
-import Language.Drasil.Printing.AST (Spec, ItemType(Flat, Nested),  
+import Language.Drasil.Printing.AST (Spec (Ch), ItemType(Flat, Nested),
```

Comment:
Minor linting thing (cuz I _love_ finding those 😅) is that there shouldn't be a space here
```suggestion
import Language.Drasil.Printing.AST (Spec(Ch), ItemType(Flat, Nested),
```